### PR TITLE
changes for self-hosted runner

### PR DESCRIPTION
**Pull request by dvviktordelev**

### CODE REVIEW

File: .github/actions/build-node/action.yml [hunk 1]
```diff
@@ -29,6 +29,10 @@ inputs:
   run_webpack:
     default: false
     required: false
+  disable_cache:
```
Comment (puehringer) [inline .github/actions/build-node/action.yml:32]: There are many usages of `disable_cache`, but I think the majority is commented out. They just pollute this PR and make it harder to digest. Can't we get rid of it?

Comment (dvviktordelev) [reply .github/actions/build-node/action.yml]: Is removed

File: .github/actions/build-node/action.yml [hunk 2]
```diff
@@ -71,8 +75,8 @@ runs:
       run: yarn install --no-immutable --inline-builds
       shell: bash
     - name: Save yarn cache
-      uses: actions/cache/save@v3
       if: steps.install.outcome == 'success'
+      uses: actions/cache/save@v3
```
Comment (puehringer) [inline .github/actions/build-node/action.yml:79]: Unnecessary line change

File: .github/workflows/build-node-python.yml [hunk 3]
```diff
@@ -160,7 +166,6 @@ jobs:
         with:
           python_version: ${{ secrets.PYTHON_VERSION || env.PYTHON_VERSION }}
           github_ro_token: ${{ github.event.repository.private == true && secrets.DATAVISYN_BOT_REPO_TOKEN || github.token }}
-
```
Comment (puehringer) [inline .github/workflows/build-node-python.yml:163]: Unnecessary line change

File: .github/workflows/build-node-python.yml [hunk 4]
```diff
@@ -230,11 +252,16 @@ jobs:
           install: false
           # TODO: Create customizable variables for these?
           start: |
+            pg_isready -d db -h ${{ env.POSTGRES_HOSTNAME }} -p ${{ env.POSTGRES_PORT }} -U admin
             yarn start dev_server_only=true
             make start
           wait-on: "http://localhost:8080, http://localhost:9000/health"
-          env: ${{ secrets.CYPRESS_ENV }}
-          spec: ${{ inputs.cypress_spec_param }}
+          #env: ${{ secrets.CYPRESS_ENV }}
```
Comment (puehringer) [inline .github/workflows/build-node-python.yml:259]: Can these commented out lines be removed?

**Pull request merged**